vscode/README.md

```diff
@@ -0,0 +1,64 @@
+# WSO2 Ballerina Integrator
+## 1.0.0
+
+Ballerina Integrator VSCode Extension is a tool that could be used alongside the Ballerina programming language to ease the creation of integration projects. This extension provides templates for integration use cases that could be used to create and start new projects. It also contains code snippets for commonly used integration scenarios.
+
+## User Guide
+
+### Requirements
+  - Visual Studio Code
+  - Ballerina (v1.0.0)
+  - Ballerina VSCode Extension (v1.0.0)
+
+### Installation
+* **Using the VSCode Marketplace**
+
+    Install the `Ballerina Integrator` VSCode Extension by WSO2.
+*Please note that this extension has not been released to the VSCode Marketplace yet.*
```

Comment:
```suggestion
*Please note that this extension has not been released to the VS Code Marketplace yet.*
```
Can we fix everywhere? It is normally written as "VS Code".

Reply:
Done